.travis/environments/travis/postgresql.yml

```diff
@@ -1,11 +1,8 @@
 DEFAULT_POSTGRESQL_PASSWORD: commcarehq
 DEFAULT_POSTGRESQL_USER: commcarehq
 SEPARATE_SYNCLOGS_DB: False
+SEPARATE_FORM_PROCESSING_DBS: False
 
-postgresql_dbs:
-  - name: commcarehq_ucr
-    django_alias: ucr
-    django_migrate: False
-  - name: "{{commcarehq_main_db_name}}"
-    django_alias: default
-  - name: "{{ formplayer_db_name }}"
+dbs:
+  synclogs: null
+  form_processing: null
```

Comment:
what's the reason for having this `null` as well as specifying `SEPARATE_X_DB`. They seem to communicate same information. It is just a check to make sure that's the intention?

Reply:
Yeah, it was originally just sort of a bridge, but I'm torn as to whether removing the explicit `SEPARATE_*` variables makes this better or just more error prone, so I decided to leave it as is and wait for inspiration. I guess I could make it only look at the `SEPARATE_*` variables and make `synclogs`/`form_processing` default to `null`, but then if you specified something there you'd still want it to fail, so that doesn't seem like much of a gain either.

I'm inclined to keep it as it is, but I'm not so attached to it and wouldn't mind having something better.